sql/ddl.sql

```diff
@@ -1666,3 +1666,110 @@ VALUES ('Bruges når et punkt og tilhørende geometri nedlægges.', 'punkt_nedla
 
 INSERT INTO eventtype (beskrivelse, event, eventtypeid)
 VALUES ('Bruges til at tilføje fritekst-kommentarer til sagen i tilfælde af at der er behov for at påhæfte sagen yderligere information, som ikke passer i andre hændelser. Bruges fx også til påhæftning af materiale på sagen.', 'kommentar', 9);
+
+-- udvidelse - flyttes "på plads" senere
+-------------------------------------------------------
+
+-- PUNKTGRUPPE
+-------------------------------------------------------
+-- Usecases:
+-- * Opret punktgruppe ud fra n punktid'er
+-- * Tilføj punkter til punktgruppe, via punktgruppe_punkt
+-- * Punkter fjernes IKKE fra punktgruppe
+-- * Ingen versionering - punktgruppe kan ift reg.fra/til og sagsevents
+--   oprettes og lukkes, ikke opdaterres
+CREATE TABLE punktgruppe (
+  objektid INTEGER GENERATED ALWAYS AS IDENTITY (
+    START WITH
+    1 INCREMENT BY 1 ORDER NOCACHE
+  ) PRIMARY KEY,
+
+  registreringfra TIMESTAMP WITH TIME ZONE NOT NULL,
+  registreringtil TIMESTAMP WITH TIME ZONE,
+  sagseventfraid VARCHAR2(36) NOT NULL,
+  sagseventtilid VARCHAR2(36),
+
+  jessenpunktid VARCHAR2(36) NOT NULL, -- Det fastholdte punkt i gruppen
```

Comment:
Hvis dette er 'navnet' på tidsserien, er det så ikke en dublet, da det (navnet) indgår SRID-ID'et for koordinaten til Jessenpunktet?

Reply:
Det er ikke "navnet" på tidsserien. Her drejer det sig udelukkende om punktgruppen, som principielt ikke har kendskab til tidsserien. Det er måske nemmere at betragte det som et "hovedpunkt" i punktgruppen. 

Mht SRID, den information der i dag er lagret i koordinaterne som fx srid=`TS:81012` erstattes af en række i `tidsserie`-tabellen samt tilhørende rækker i `tidsserie_koordinat`-tabellen. Men det er jo også muligt at lave en tidsserie ud fra en række DVR90-koter eller igb14-koordinater, og i så fald er det ikke helt lige så oplagt at sætte koordinatens SRID til en  `TS:whatever`. Faktisk mener jeg der er god grund til lade være hvis vi kan, da det gør det lettere at håndtere transformation af koordinaterne i tilfælde hvor det måtte være nyttigt.

Reply:
> (...) det er jo også muligt at lave en tidsserie ud fra en række DVR90-koter eller igb14-koordinater, og i så fald er det ikke helt lige så oplagt at sætte koordinatens SRID til en `TS:whatever`. (...)

Tak. Jeg ser, at min indskydelse var for tæt knyttet til AS-IS her, men dét, du uddyber med her, er jo præcis årsagen til, at vi laver ændringen i datamodellen: så vi ikke skal blande 'rigtige' reference-systemer sammen med en 'etiket' på en tidsserie:)